rps.js

```diff
@@ -0,0 +1,25 @@
+
+const playGame= function(yourMove)
+{
+    const moves = ["rock","paper","scissors"];
+    const randomMove = moves[Math.floor(Math.random() * moves.length)];
+    if(yourMove == randomMove)
+    {
+        return "Tie";
+
+    } else if((yourMove === "paper" && moves[randomMove]=== "rock") || (yourMove =="rock" && randomMove=="paper") || (yourMove=="scissors" && randomMove=="paper")||(yourMove=="rock" && randomMove=="scissors"))
```

Comment:
one of the conditions is wrong, you have two conditions for rock wining.
remove the condition that user player rock and computer plays paper, and your logic will be correct.